rest_framework_simplejwt/authentication.py

```diff
@@ -23,6 +23,9 @@ class JWTAuthentication(authentication.BaseAuthentication):
     """
     www_authenticate_realm = 'api'
 
+    def __init__(self):
+        self.user_model = get_user_model()
```

Comment:
Actually, did you check that the `authentication.BaseAuthentication` class doesn't have an `__init__` method that needs to be called?  In other words, should this `__init__` method maybe look like this?
```suggestion
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
        self.user_model = get_user_model()
```

Reply:
@davesque according to [the source code](https://github.com/encode/django-rest-framework/blob/master/rest_framework/authentication.py#L33) `authentication.BaseAuthentication` doesn’t have `__init__ `

Reply:
@davesque i already updated

Reply:
Yeah, I suppose one could be added in the future.  So best to play it safe.